doc/acl/Everflow-test-plan.md

```diff
@@ -0,0 +1,1157 @@
+
+- [Overview](#Overview)
+  - [Scope](#Scope)
+  - [Summary of the existing everflow test plan](#Summary-of-the-existing-everflow-test-plan)
+  - [Extend the test plan to cover both ingress and egress mirroring](#Extend-the-test-plan-to-cover-both-ingress-and-egress-mirroring)
+    - [What new enhancements need to be covered?](#What-new-enhancements-need-to-be-covered)
+      - [Egress ACL table](#Egress-ACL-table)
+      - [Egress mirroring](#Egress-mirroring)
+    - [Some existing areas not covered by the existing scripts](#Some-existing-areas-not-covered-by-the-existing-scripts)
+      - [ACL rule for matching "IN_PORTS"](#ACL-rule-for-matching-IN_PORTS)
+      - [IPv6 everflow](#IPv6-everflow)
+    - [How to extend the testing](#How-to-extend-the-testing)
+  - [Test configurations](#Test-configurations)
+    - [ACL table configurations](#ACL-table-configurations)
+  - [Related **DUT** CLI commands](#Related-DUT-CLI-commands)
+    - [`sonic-cfggen`](#sonic-cfggen)
+    - [`acl-loader`](#acl-loader)
+    - [`aclshow`](#aclshow)
+    - [`config mirror_session`](#config-mirror_session)
```

Comment:
could you add config acl table/rule session? I would like to have some CLI supports for table/rule creation/removal

Reply:
Sure, added in the new commit.